adoc/admin-security-certificates.adoc

```diff
@@ -149,6 +149,12 @@ We use cert-exporter to monitor nodes' on-host certificates and addons' secret c
 helm repo add suse https://kubernetes-charts.suse.com
 helm install suse/cert-exporter --name ${RELEASE_NAME}
 ----
+Or if you have selected the Helm 3 alternative <<helm-tiller-install>>:
+[source,bash]
+----
+helm repo add suse https://kubernetes-charts.suse.com
+helm install suse/cert-exporter ${RELEASE_NAME}
```

Comment:
In Helm3, the name should come before chart.

Reply:
done